src/Compilers/Core/CodeAnalysisTest/Emit/EmitBaselineTests.cs

```diff
@@ -20,17 +20,22 @@ public void CreateInitialBaseline_Errors()
             var peReader = peModule.Module.PEReaderOpt;
 
             var mdBytes = peReader.GetMetadata().GetContent();
-            var mdBytesHandle = GCHandle.Alloc(mdBytes.DangerousGetUnderlyingArray(), GCHandleType.Pinned);
-            var mdModule = ModuleMetadata.CreateFromMetadata(mdBytesHandle.AddrOfPinnedObject(), mdBytes.Length);
+            unsafe
+            {
+                fixed (byte* ptr = mdBytes.AsSpan())
+                {
+                    var mdModule = ModuleMetadata.CreateFromMetadata((IntPtr)ptr, mdBytes.Length);
```

Comment:
```suggestion
                    using var mdModule = ModuleMetadata.CreateFromMetadata((IntPtr)ptr, mdBytes.Length);
```

Reply:
There's nothing to dispose here, we're using a pointer for the metadata